src/helpers/WalkHelper.php

```diff
@@ -103,9 +102,8 @@ public static function spawnTasks($type, $elements, $settings = [], $valParam =
 			{
 				$settings = is_array($settings) ? $settings : [];
 				$settings[$valParam] = $val;
-				$task = Craft::$app->getTasks()->createTask(['type' => $type, 'settings' => $settings]);
-				if ($task instanceof MissingTask)
```

Comment:
@michaelrog this is the only thing I'm not 100% sure how to "update"; I couldn't find a Craft 3 equivalent.

Reply:
@jalendport This is a vestige of the Tasks service that shipped with Craft 3 in the beta. `Tasks:createTask()` used to return a Task object. `Queue:push()` returns a job ID or `null`. So instead of checking for a `MissingTask`, we can now just check to make sure the return isn't `empty()`. If we get an ID back, the job was queued successfully. Otherwise, we return that `false` as an indicator to the caller that we were unable to queue the job.

Reply:
Ah so it's a Craft 3 beta remnant. Explains why I couldn't find it in the Craft 2 docs or codebase. I'll add that `empty()` conditional like you said 👍